nuxeo-opencmis-tests/src/test/java/org/nuxeo/ecm/core/opencmis/impl/CmisSuiteSession.java

```diff
@@ -417,6 +417,13 @@ public void testCreateHiddenRelation() throws Exception {
         assertEquals(id2, rel.getTargetId().getId());
     }
 
+    @Test
+    public void testCreatePath() throws Exception {
+        session.createPath("/testfolder1/newlevel/newfolder", "Folder");
+        session.getObjectByPath("/testfolder1/newlevel");
```

Comment:
What's the return datatype?  Is it better to `assertNotNull()`?

Reply:
It never returns `null`. It throws if it doesn't find the doc, so this is enough for this particular test.

Reply:
It would have been useful to add a comment to say the code assumes an unsuccessful call throws an exception